src/implementation/Client/GRPCClient/GRPCClient.ts

```diff
@@ -65,4 +65,8 @@ export default class GRPCClient implements IClient {
   async stop(): Promise<void> {
     this.client.close();
   }
+
+  async start(): Promise<void> {
+    this.client.close();
```

Comment:
Should we `start` here?

Reply:
Also, should we add this to HTTP Client too?

Reply:
Goosd catch! It's not called here, so I am removing it